Library/Homebrew/test/.rubocop_todo.yml

```diff
@@ -10,168 +10,167 @@
 # Configuration parameters: Max.
 RSpec/ExampleLength:
   Exclude:
-    - 'rubocops/patches_spec.rb'
+    - "rubocops/patches_spec.rb"
```

Comment:
I assume this is an editor thing. I'm not opposed to it but it does make the diffs a little harder to read (can't easily tell if lines were added/removed). Not a huge deal, but I'd rather we be consistent about this in the future. I feel like I've seen it flip-flop several times.